packages/cubejs-cockroach-driver/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@cubejs-backend/cockroach-driver",
+  "description": "Cube.js Cockroach database driver",
+  "author": "Cube Dev, Inc.",
+  "version": "0.30.25",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cube-js/cube.js.git",
+    "directory": "packages/cubejs-cockroach-driver"
+  },
+  "engines": {
+    "node": "^12.0.0 || ^14.0.0 || >=16.0.0"
+  },
+  "files": [
+    "dist/src",
+    "index.js"
+  ],
+  "main": "index.js",
+  "typings": "dist/src/index.d.ts",
+  "scripts": {
+    "build": "rm -rf dist && npm run tsc",
+    "tsc": "tsc",
+    "watch": "tsc -w",
+    "integration": "npm run integration:cockroach",
+    "integration:cockroach": "jest --verbose --maxWorkers=1 dist/test",
```

Comment:
🎵 
as far as I can see maxWorkers required cause of test containers can down environment from sibling process